src/Gui/Contexts/MainContext.luau

```diff
@@ -0,0 +1,63 @@
+local Hoarcekat = script.Parent:FindFirstAncestor("Hoarcekat")
+
+local ReactRoblox = require(Hoarcekat.Packages.ReactRoblox)
+local React = require(Hoarcekat.Packages.React)
+
+local Context = React.createContext({
+	selectedStory = nil,
+	setSelectedStory = function() end,
+} :: {
+	selectedStory: Instance?,
+	setSelectedStory: (any) -> (),
+})
```

Comment:
1. Under Types require
2. Follow the pattern in the [Contexts and Luau](https://blog.boyned.com/articles/things-i-learned-using-react/) section. It guarantees the best types.